examples/dkg_test.go

```diff
@@ -21,8 +23,24 @@ the "happy" path where each node does its job correctly.
 */
 func Test_Example_DKG(t *testing.T) {
 
-	// number of nodes
-	n := 3
+	// DKG scales exponentially, the following command prints the duration [ns]
+	// of this test case with an increasing number of nodes. The resulting plot
+	// should illustrate an exponential growth.
+	//
+	// for (( i=1; i<30; i++ )); do
+	//   start=`gdate +%s%N`
+	//   NUM_NODES=$i go test -run Test_Example_DKG >/dev/null
+	//   duration=$(( `gdate +%s%N` - start ))
+	//   echo $duration
+	// done
+	//
+	var nStr = os.Getenv("NUM_NODES")
```

Comment:
`os.Getenv` in a test? Sounds really strange to me. What about a benchmark? Or just a for-loop in the test?

The only thing I do sometimes is to check for `testing.Short()`. In this case, it could be that the full test has `n := 7`, but using `testing.Short()` it uses only `n := 3`.

Reply:
This provides a quick way to do a benchmark, as described in the comment above. As this scenario is a small demonstration of DKG I find it also interesting to include a word about the complexity. This is an opt-in functionality and the test is stable in its default behaviour. 

For the record:

![Time vs number of nodes](https://user-images.githubusercontent.com/1918021/80703129-7fd30800-8ae2-11ea-9230-b15d2a32986e.png)